### DISCUSSION TITLE
这个要怎么安装呀 直接上传 音乐 和地址 都用不了

### DISCUSSION THREAD

**Discussion by losie** [imsyy/home#73] [Announcements]
求大佬指点

**Comment by imsyy** [comment]
具体是哪里用不了？控制台有报错吗

**Reply by losie** [reply]
谢谢大佬 回复 Uncaught TypeError: Failed to resolve module specifier "vue". Relative references must start with either "/", "./", or "../".

**Reply by losie** [reply]
控制台报错 Uncaught TypeError: Failed to resolve module specifier "vue". Relative references must start with either "/", "./", or "../".

**Reply by imsyy** [reply]
你需要先安装依赖
具体请看说明文档 https://github.com/imsyy/home#%E9%83%A8%E7%BD%B2

**Reply by imsyy** [reply]
或者这里 https://github.com/imsyy/home/discussions/61#discussioncomment-4840642

**Reply by losie** [reply]
知道啦 是我没理解 谢谢！
